lib/services/local/users.go

```diff
@@ -1246,17 +1246,18 @@ func (s *IdentityService) GetOIDCConnectors(ctx context.Context, withSecrets boo
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	connectors := make([]types.OIDCConnector, len(result.Items))
-	for i, item := range result.Items {
+	var connectors []types.OIDCConnector
+	for _, item := range result.Items {
 		conn, err := services.UnmarshalOIDCConnector(item.Value, services.WithExpires(item.Expires), services.WithRevision(item.Revision))
 		if err != nil {
-			return nil, trace.Wrap(err)
+			logrus.WithError(err).Errorf("Error unmarshaling OIDC Connector: %s", item.Key)
```

Comment:
Go full structured logging:

```suggestion
			logrus.
				WithError(err).
				WithField("key", item.Key).
				Error("Error unmarshaling OIDC connector")
```

This may seem like a nit, but it allows folks logging in JSON to better index and query logs.

Same for others.